test/application/use.js

```diff
@@ -66,6 +66,6 @@ describe('app.use(fn)', function(){
   it('should throw error for generator', function(){
     const app = new Koa();
 
-    (() => app.use(function *(){})).should.throw('Support for generators has been removed. Use Promises or wrap your generator with co.wrap');
+    (() => app.use(function *(){})).should.throw('Support for generators has been removed. See the documentation for examples of how to convert old middleware https://github.com/koajs/koa');
```

Comment:
for the test, you could also just test that a substring exists so you don't have to make these two strings EXACTLY the same